foundation/plugins/org.jboss.tools.foundation.ui/src/org/jboss/tools/foundation/ui/credentials/internal/CredentialMessages.properties

```diff
@@ -18,3 +18,14 @@ SelectDomain=Please select a domain.
 UsernameCannotBeBlank=Username cannot be blank.
 PasswordCannotBeBlank=Password cannot be blank.
 UsernameAlreadyExists=Username {0} already exists for domain {1}.
+
+# For use in the credential prompter
+OK=OK
+Cancel=Cancel
+CredentialPrompterTitle=Password Required
+DescriptionSectionTitle=Description
+DescriptionSectionContent=<form><p>A password is required for the username {0} and the credential domain {1}</p></form>
```

Comment:
This was copied from the foundation.security.linux plugin.  I attempted to remove the form tags, but I got UI errors, specifically exceptions were thrown. It seems the widget in use in the dialog requires the form tag. Rather than mess around with the structure of the dialog, I chose to keep it more closely aligned in case one day we can unify the two classes with some sort of abstract superclass.